subsys/bluetooth/controller/ll_sw/nordic/lll/lll_sync_iso.c

```diff
@@ -275,16 +294,56 @@ static int prepare_cb_common(struct lll_prepare_param *p)
 	/* Initialize stream current */
 	lll->stream_curr = 0U;
 
-	const bool is_sequential_packing = (lll->bis_spacing >= (lll->sub_interval * lll->nse));
-
 	/* Skip subevents until first selected BIS */
 	stream_handle = lll->stream_handle[lll->stream_curr];
 	stream = ull_sync_iso_lll_stream_get(stream_handle);
 	if ((stream->bis_index != lll->bis_curr) &&
 	    (stream->bis_index <= lll->num_bis)) {
 		/* First selected BIS */
 		lll->bis_curr = stream->bis_index;
+	}
+
+	const bool is_sequential_packing = (lll->bis_spacing >= (lll->sub_interval * lll->nse));
+
+#if defined(CONFIG_BT_CTLR_SYNC_ISO_SLOT_WINDOW_JITTER)
+	uint8_t skipped = 0U;
+
+	if (p->ticks_drift) {
+		uint8_t skipped_bis;
+		uint32_t drift_us;
 
+		drift_us = HAL_TICKER_TICKS_TO_US(p->ticks_drift);
+
+		/* FIXME: Add implementation to support interleaved packing */
+		LL_ASSERT(is_sequential_packing);
```

Comment:
Is an assert the right option here? 

I'm wondering if we should just reject the sync request / terminate the BIG sync instead